proto/wechaty/puppet/message.proto

```diff
@@ -170,3 +170,11 @@ message MessageRecallRequest {
 message MessageRecallResponse {
   bool success = 1;
 }
+message MessageForwardRequest{
+  string conversation_id = 1;
+  MessageType type = 2;
```

Comment:
Please use `message_id` to replace the `type` and `payload` because the `MessageType` and the `payload` can be represented by the `messageId` with the puppet service backend which can get the payload by the id easily.

So please align this gRPC service with our wechaty-puppet design here:

https://github.com/wechaty/wechaty-puppet/blob/3d009a0d1482c78ed945ab983c070e7a4fa4d31b/src/puppet.ts#L935-L938

My suggestion is:

```
message MessageForwardRequest {
  string conversation_id = 1;
  string message_id = 2;
}
```